src/gateways/createArchiveLoader.ts

```diff
@@ -66,6 +67,7 @@ export default (url: string) => {
         page: (pageId: string) => ({
           cached: () => pageCache.get(`${bookRef}:${pageId}`),
           load: () => pageLoader(`${bookRef}:${pageId}`),
+          url: () => `${url}/contents/${bookRef}:${pageId}`,
```

Comment:
i wanted you to dry this up, so `${bookRef}:${pageId}` is not in 3 places and `/contents` is not in two